crypto/fipsmodule/rand/urandom.c

```diff
@@ -252,6 +256,13 @@ static void init_once(void) {
   return;
 #endif
 
+#if defined(OPENSSL_OPENBSD)
+  // To get system randomness on OpenBSD we use |arc4random_buf| function
+  // which is recommended to use for C APIs rather then /dev/random.
```

Comment:
```suggestion
  // which is recommended to use for C APIs rather then /dev/urandom.
```

Leaving this note so that I can fix it after I get other comments.